src/components/missions/MissionItem.jsx

```diff
@@ -16,8 +16,11 @@ const MissionItem = ({ id, name, description, status }) => {
       <td style={{ width: '10%' }}>{name}</td>
       <td style={{ width: '60%' }}>{description}</td>
       <td className="text-center" style={{ width: '15%' }}>
-        <Badge bg="secondary">Not A Member</Badge>
-        {/* <Badge bg="primary">Active Member</Badge> */}
+        {!status ? (
+          <Badge bg="secondary">Not A Member</Badge>
+        ) : (
+          <Badge bg="primary">Active Member</Badge>
+        )}
```

Comment:
I think it is more efficient to do a variable(But I am not sure we could argue):
```javascript
const button = !status 
  ? { class: "secondary", text: "Not a Member" } 
  : { class: "secondary", text: "Active Member" };
```
.
.
.
```javascript
<Badge bg={button.class}>{button.text}</Badge>
```

Reply:
Thank you for your review @Leboroz, but I think it's not a big deal because React [doc](https://reactjs.org/docs/conditional-rendering.html) uses the same way (Inline If-Else with Conditional Operator)

Maybe your way is more readable but I can use it only with the `badges`. But I can't use it with the `buttons` because I have `events` on these buttons (onClick).

I agree that your way is more readable, so I will implement it on the `badges`. 💯